components/pkg-export-helm/src/deps.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2018 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use failure::SyncFailure;
+use handlebars::Handlebars;
+use clap;
+use std::io::Write;
+use std::path::Path;
+use std::process::Command;
+
+use common::ui::{UI, Status};
+use export_docker::Result;
+
+pub const DEFAULT_OPERATOR_VERSION: &'static str = "0.5.1";
+pub const OPERATOR_REPO_URL: &'static str = "https://kinvolk.github.io/habitat-operator\
+                                             /helm/charts/stable/";
+
+// Helm requirements.yaml template
+const DEPSFILE: &'static str = include_str!("../defaults/HelmDeps.hbs");
+
+pub struct Deps {
+    operator_version: String,
+    update: bool,
+}
+
+impl Deps {
+    pub fn new_for_cli_matches(matches: &clap::ArgMatches) -> Self {
+        Deps {
+            operator_version: matches
+                .value_of("OPERATOR_VERSION")
+                .unwrap_or(DEFAULT_OPERATOR_VERSION)
```

Comment:
Given that we define a default, is this `unwrap_or` needed?

Reply:
I am not sure I understand the question.  `unwrap_of` gives you the default here if user didn't pass that argument on the CLI.

Reply:
Ugh, sorry about that... I think my brain must have been off when I typed that. I read it and understood it as `unwrap`, even though I typed `unwrap_or`. 

You are, of course, correct.

![tenor-151105279](https://user-images.githubusercontent.com/207178/36430410-1caf4670-1623-11e8-842c-6b94ddbfe8be.gif)

Reply:
No worries, I wouldn't add an unwrap() call without a comment explaining why it's OK. Now a days, I try to use expect() instead of unwrap() so it's easy to trace the reasons.